Edit.py

```diff
@@ -0,0 +1,38 @@
+import PrintList
+
+def edit(to_Do):
+  # Print List
+  PrintList.print_list(to_Do)
+
+  X = input("[Press X if you want to exit]").upper()
+
+  if X == 'x':
```

Comment:
We raun `upper()` on line 7 so does this need to compare to an uppercase X? Or does lowercase x still work?